extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -184,7 +210,10 @@ export class AmpPanZoom extends AMP.BaseElement {
 
   /** @override */
   isLayoutSupported(layout) {
-    return layout == Layout.FIXED || layout == Layout.FILL;
```

Comment:
Should this be `src/layout.js#isLayoutSizeDefined`? Would all of those layouts make sense for `amp-pan-zoom` as well? The current list makes me think `amp-pan-zoom` should behave similarly to other components that use size defined layout, like `amp-img` and `amp-carousel`.

https://github.com/ampproject/amphtml/blob/ae12962f3ad8b4cafa3723557a82c009ebc093fe/src/layout.js#L143-L156

Reply:
I think this is where we eventually want to get. At its current stage, it probably makes sense to add approved layouts as I verify that they work.

Reply:
Cool, makes sense